vcx/libvcx/src/settings.rs

```diff
@@ -55,6 +56,10 @@ pub static DEFAULT_WALLET_KEY: &str = "foobar1234";
 pub static DEFAULT_THREADPOOL_SIZE: usize = 8;
 pub static MASK_VALUE: &str = "********";
 pub static DEFAULT_WALLET_KEY_DERIVATION: &str = "ARGON2I_INT";
+pub static DEFAULT_PAYMENT_PLUGIN: &str = "libnullpay.so";
+pub static DEFAULT_PAYMENT_INIT_FUNCTION: &str = "nullpay_init";
+pub static DEFAULT_PAYMENT_METHOD: &str = "null";
```

Comment:
If somebody wanted to initialize the "libsovtoken" payment plugin would all three have to be configured?  Are there configuration options for all three different values?

Reply:
Yes, if libsovtoken is to be used these should be changed.

Config options are not available because only method is something that needs to be configured for libvcx, these DEFAULTS_* are just for testing purposes.